app/src/main/java/org/oppia/android/app/drawer/DeveloperOptionsStarter.kt

```diff
@@ -0,0 +1,9 @@
+package org.oppia.android.app.drawer
+
+import android.content.Context
+import android.content.Intent
+
+/** Interface to create intent for [DeveloperOptionsActivity]. */
+interface DeveloperOptionsStarter {
+  fun createIntent(context: Context, internalProfileId: Int): Intent
```

Comment:
Why do we need this? Why can't we just call `DeveloperOptionsActivity.createDeveloperOptionsActivityIntent(context, internalProfileId)` directly similar to our current code base ?

Reply:
It is necessary to follow this approach as for non-debug builds, we won't be including DeveloperOptionsActivity and all the related files to it in our build. So, if we follow the conventional approach the app will throw a build time error. Having this Starter provides Optional binding for the intent which means if the Dev modules are not found in the build it won't throw an error. It will just treat it as if it is not present.

Reply:
Makes sense. Thanks.